test-result-summary-client/src/Build/Summary/ResultSummary.jsx

```diff
@@ -118,10 +121,12 @@ export default function ResultSummary() {
                     }
                     // use non-capture group to ignore words evaluation and release if present
                     const regex =
-                        /^jdk(\d+).?(?:-evaluation|-release)?-(\w+)-(\w+)-(\w+)/i;
+                        /^jdk(\d*).?(?:-evaluation|-release)?-(\w+)-(\w+)-(\w+)/i;
                     const tokens = buildName.match(regex);
                     if (Array.isArray(tokens) && tokens.length > 4) {
-                        jdkVersion = tokens[1];
+                        if (!jdkVersion && tokens[1]) {
+                            jdkVersion = tokens[1];
+                        }
```

Comment:
I think we should use jdkVersion if the build name contains it. If it does not contain jdkVersion, then use the value fron its parent build. Since the default jdkVersion is set above, we just need to reset if `tokens[1]` has value.

```
if  (tokens[1]) {
    jdkVersion = tokens[1];
}
```